content/en/docs/next/op-guide/supported-platform.md

```diff
@@ -8,29 +8,29 @@ description: etcd support status for common architectures & operating systems
 
 The following table lists etcd support status for common architectures and operating systems:
 
-| Architecture | Operating System | Status       | Maintainers                 |
+| Architecture | Operating System | Support Tier | Maintainers                 |
 | ------------ | ---------------- | ------------ | --------------------------- |
-| amd64        | Darwin           | Experimental | etcd maintainers            |
-| amd64        | Linux            | Stable       | etcd maintainers            |
-| amd64        | Windows          | Experimental |                             |
-| arm64        | Linux            | Experimental | @glevand                    |
-| arm          | Linux            | Unstable     |                             |
-| 386          | Linux            | Unstable     |                             |
-| ppc64le      | Linux            | Stable       | etcd maintainers, @mkumatag |
+| amd64        | Linux            | Tier-1       | etcd maintainers            |
+| arm64        | Linux            | Tier-2       | @gyuho, @glevand            |
+| amd64        | Darwin           | Tier-3       |                             |
+| amd64        | Windows          | Tier-3       |                             |
+| arm          | Linux            | Tier-3       |                             |
+| 386          | Linux            | Tier-3       |                             |
+| ppc64le      | Linux            | Tier-3       | @mkumatag                   |
 
-* etcd-maintainers are listed in https://github.com/etcd-io/etcd/blob/master/MAINTAINERS.
+*etcd-maintainers are listed in [MAINTAINERS](https://github.com/etcd-io/etcd/blob/master/MAINTAINERS).*
 
-Experimental platforms appear to work in practice and have some platform specific code in etcd, but do not fully conform to the stable support policy. Unstable platforms have been lightly tested, but less than experimental. Unlisted architecture and operating system pairs are currently unsupported; caveat emptor.
+Tier-1 platforms are fully supported by etcd maintainers and required to pass all tests including functional tests. Tier-2 platforms appear to work in practice but may have some platform specific code in etcd and not fully conform to the stable support policy. To qualify for Tier-2, the platform must pass integration and end-to-end tests in CI (see [github PR](https://github.com/etcd-io/etcd/pull/12928) for adding arm64). Tier-3 platforms or unlisted architectures are either lightly tested or have no testing in place, thus unstable and currently unsupported; caveat emptor.
```

Comment:
A quick copy-edit pass here. I'd also suggest using bold to set off the tier descriptions to help scannability.
```suggestion
*Tier-1* platforms are fully supported by etcd maintainers and are required to pass all tests including functional ones. *Tier-2* platforms appear to work in practice but may have some platform specific code in etcd and may not fully conform to the stable support policy. To qualify for Tier-2, the platform must pass integration and end-to-end tests in the CI system (see [GitHub PR](https://github.com/etcd-io/etcd/pull/12928) for adding arm64). *Tier-3* platforms, or unlisted architectures, are either lightly tested or have no testing in place, and are thus unstable and currently unsupported; caveat emptor.
```